src/Oro/Bundle/SearchBundle/Migrations/Schema/v1_2/OroSearchBundle.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Oro\Bundle\SearchBundle\Migrations\Schema\v1_2;
+
+use Doctrine\DBAL\Schema\Schema;
+
+use Doctrine\ORM\EntityManager;
+use Oro\Bundle\MigrationBundle\Migration\Migration;
+use Oro\Bundle\MigrationBundle\Migration\QueryBag;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class OroSearchBundle implements Migration , ContainerAwareInterface
+{
+    /**
+     * @var ContainerInterface
+     */
+    protected $container;
+
+
+    /**
+     * @inheritdoc
+     */
+    public function setContainer(ContainerInterface $container = null)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function up(Schema $schema, QueryBag $queries)
+    {
+        //save old avatars to new place
+        /** @var EntityManager $em */
+        $em         = $this->container->get('doctrine.orm.entity_manager');
```

Comment:
please do not use EntityManager in a migration. To get a connection the migration can implement `Oro\Bundle\MigrationBundle\Migration\ConnectionAwareInterface` interface